### PR TITLE
Docsite: fix playbooks_conditionals

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Docsite: fix playbooks_conditionals

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
`docs/docsite/rst/user_guide/playbooks_conditionals.rst`

### CODE REVIEW

**Comment by Andersson007** [comment]
ready_for_review

File: docs/docsite/rst/user_guide/playbooks_conditionals.rst [hunk 1]
```diff
@@ -141,17 +148,21 @@ Ansible always registers something in a registered variable for every host, even
...
     tasks:
-      - command: /bin/false
+      - name: Register a variable, ignore errors and continue
+        ansible.builtin.command: /bin/false
         register: result
-        ignore_errors: True
+        ignore_errors: true
 
-      - command: /bin/something
+      - name: Run only if the tak that registered the "result" variable fails
```
Comment (samccann) [inline docs/docsite/rst/user_guide/playbooks_conditionals.rst]:
```suggestion
      - name: Run only if the task that registered the "result" variable fails
```

File: docs/docsite/rst/user_guide/playbooks_conditionals.rst [hunk 2]
```diff
@@ -141,17 +148,21 @@ Ansible always registers something in a registered variable for every host, even
...
+        ansible.builtin.command: /bin/something
         when: result is failed
 
-      - command: /bin/something_else
+      - name: Run only if the tak that registered the "result" variable succeeds
```
Comment (samccann) [inline docs/docsite/rst/user_guide/playbooks_conditionals.rst]:
```suggestion
      - name: Run only if the task that registered the "result" variable succeeds
```

File: docs/docsite/rst/user_guide/playbooks_conditionals.rst [hunk 3]
```diff
@@ -141,17 +148,21 @@ Ansible always registers something in a registered variable for every host, even
...
+        ansible.builtin.command: /bin/something_else
         when: result is succeeded
 
-      - command: /bin/still/something_else
+      - name: Run only if the tak that registered the "result" variable is skipped
```
Comment (samccann) [inline docs/docsite/rst/user_guide/playbooks_conditionals.rst]:
```suggestion
      - name: Run only if the task that registered the "result" variable is skipped
```

File: docs/docsite/rst/user_guide/playbooks_conditionals.rst [hunk 4]
```diff
@@ -363,10 +396,12 @@ Then import those variables files based on the facts you gather on the hosts in
...
-        service: name={{ apache }} state=started
+      - name: Make sure apache is started
+        ansible.builtin.service:
+          name: '{{ apache }}'
+          state: started
 
-Ansible gathers facts on the hosts in the webservers group, then interpolates the variable "ansible_facts['os_family']" into a list of filenames. If you have hosts with Red Hat operating systems ('CentOS', for example), Ansible looks for 'vars/RedHat.yml'. If that file does not exist, Ansible attempts to load 'vars/os_defaults.yml'. For Debian hosts, Ansible first looks for 'vars/Debian.yml', before falling back on 'vars/os_defaults.yml'. If no files in the list are found, Ansible raises an error.
+Ansible gathers facts on the hosts in the webservers group, then interpolates the variable "ansible_facts['os_family']" into a list of filenames. If you have hosts with Red Hat operating systems (CentOS, for example), Ansible looks for C(vars/RedHat.yml). If that file does not exist, Ansible attempts to load C(vars/os_defaults.yml). For Debian hosts, Ansible first looks for C(vars/Debian.yml), before falling back on C(vars/os_defaults.yml). If no files in the list are found, Ansible raises an error.
```
Comment (samccann) [inline docs/docsite/rst/user_guide/playbooks_conditionals.rst]: C doesn't work in .rst files.

File: docs/docsite/rst/user_guide/playbooks_conditionals.rst (hunk 1 above)
Comment (Andersson007) [reply docs/docsite/rst/user_guide/playbooks_conditionals.rst]: OMG, thanks! fixed

**Comment by Andersson007** [comment]
ready_for_review

**Review by samccann — APPROVED** [review]

**Pull request merged**